src/Page.js

```diff
@@ -227,24 +242,76 @@ function getClosestHeading($, headingsSelector, element) {
 }
 
 /**
- * Records headings and keywords inside rendered page into this.headings and this.keywords respectively
+ * Checks if page.frontMatter has a valid page navigation specifier
+ * @returns boolean
  */
-Page.prototype.collectHeadingsAndKeywords = function () {
-  const $ = cheerio.load(fs.readFileSync(this.resultPath));
-  this.collectHeadingsAndKeywordsInContent($(`#${CONTENT_WRAPPER_ID}`).html(), null, false);
+Page.prototype.validPageNavigationSpecifier = function () {
```

Comment:
Rename the method name to `isPageNavigationSpecifierValid` so that it is understood that it returns a boolean (then perhaps the comment `@returns boolean` is no longer necessary?)